bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/listings/ListingsTableAccessor.java

```diff
@@ -72,6 +69,22 @@ public Listing getListingById(int listingId) {
 	/// this can be done by doing `INSERT INTO listings ... RETURNING listing_id`
 	/// and parsing the `listing_id` column it returns with `results.getString("listing_id")`
 	public void createListing(ListingWithoutId listingWithoutId) {
-		// TODO
+		String sql ="INSERT INTO listings (title, author, description, classSubject, price, condition, createdAt, saleAvailability, sellerId" + "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)";
```

Comment:
Few things here:
1. Remember our SQL column names are in `snake_case` not `camelCase`, so these variables should be changed to `snake_case`. For ex. `classSubject` -> `class_subject`
2. You're missing a closing parenthesis on the `(title, author, ..., sellerId`
3. Because of the String `+`, the query will look like `INSERT INTO ... sellerIdVALUES (` which is a syntax error. There should be a space before `VALUES`. You can get rid of the `+` to reduce confusion here, not sure if it was an accident